class.jetpack.php

```diff
@@ -7274,14 +7274,8 @@ public static function is_gutenberg_available() {
 	/**
 	 * Load Gutenberg editor blocks.
 	 *
-	 * This section meant for unstable phase of developing Jetpack's
-	 * Gutenberg extensions. If still around after Sep. 15, 2018 then
-	 * please file an issue to remove it; if nobody responds within one
-	 * week then please delete the code.
-	 *
-	 *
-	 * Loading blocks is disabled by default and enabled via filter:
-	 *   add_filter( 'jetpack_gutenberg', '__return_true' );
+	 * Loading blocks is enabled by default and may be disabled via filter:
+	 *   add_filter( 'jetpack_gutenberg', '__return_false' );
 	 *
 	 * When enabled, blocks are loaded from CDN by default. To load locally instead:
```

Comment:
Shouldn't this say "not loaded from CDN" by default now?

Reply:
👍 Thanks for highlighting this.

I've removed the comment, I don't think it makes sense at this level. The filter is documented a bit lower. The CDN is no longer enabled by default and there doesn't seem to be much reason to enable it.